src/select/mini.rs

```diff
@@ -381,9 +381,10 @@ where
                 format!("width: {w}px;")
             }
         });
-        let style_height = ctx.props()
-                .top_height
-                .map_or_else(String::new, |h| format!("height: {h}px;"));
+        let style_height = ctx
+            .props()
+            .top_height
+            .map_or_else(String::new, |h| format!("height: {h}px;"));
```

Comment:
Why has this been changed? I guess If we had been all in the same settings, it wouldn't have happened.

Reply:
I ran below

```
cargo fmt -- --config group_imports=StdExternalCrate 
```

which caused the change in the format.

Reply:
It means the person who requested the previous PR, #145, didn't have the same settings in her/his VS Code. We need to track it down and ask her/him to get it fixed.

Reply:
@jinjoolee07 Could you check above comment please, and setup your VS code settings accordingly?

Reply:
@sophie-cluml checked all my settings, and everything is up to date!